frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -63,15 +63,16 @@ def make_notification_logs(doc, users):
 	from frappe.social.doctype.energy_point_settings.energy_point_settings import is_energy_point_enabled
 
 	for user in users:
-		if frappe.db.exists('User', {"email": user, "enabled": 1}):
+		username = frappe.db.exists("User", {"email": user, "enabled": 1})
```

Comment:
```suggestion
		username = frappe.db.get_value("User", {"email": user, "enabled": 1})
```
When the value is used for anything other than existence check it should be `get_value` for correct semantics.